.github/workflows/plugins.yml

```diff
@@ -105,30 +105,64 @@ jobs:
     - name: Ensure specs still run
       run: bundle exec rake spec
 
-  # run_solargraph_rspec_specs:
-  #   # check out solargraph-rspec as well as this project, and point the former to use the latter as a local gem
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #   - uses: actions/checkout@v3
-  #   - name: clone https://github.com/lekemula/solargraph-rspec/
-  #     run: |
-  #      cd ..
-  #      git clone https://github.com/lekemula/solargraph-rspec.git
-  #      cd solargraph-rspec
-  #   - name: Set up Ruby
-  #     uses: ruby/setup-ruby@v1
-  #     with:
-  #       ruby-version: '3.0'
-  #       bundler-cache: false
-  #   - name: Install gems
-  #     run: |
-  #      cd ../solargraph-rspec
-  #      echo "gem 'solargraph', path: '../solargraph'" >> Gemfile
-  #      bundle install
-  #   - name: Run specs
-  #     run: |
-  #       cd ../solargraph-rspec
-  #       bundle exec rake spec
+  run_solargraph_rspec_specs:
+    # check out solargraph-rspec as well as this project, and point the former to use the latter as a local gem
+    runs-on: ubuntu-latest
+    env:
+      SOLARGRAPH_CACHE: ${{ github.workspace }}/../solargraph-rspec/vendor/solargraph/cache
+      BUNDLE_PATH: ${{ github.workspace }}/../solargraph-rspec/vendor/bundle
+    steps:
+    - uses: actions/checkout@v3
+    - name: clone https://github.com/lekemula/solargraph-rspec/
+      run: |
+       cd ..
+       # git clone https://github.com/lekemula/solargraph-rspec.git
+
+       # pending https://github.com/lekemula/solargraph-rspec/pull/30
+       git clone https://github.com/apiology/solargraph-rspec.git
+       cd solargraph-rspec
+       git checkout reset_closures
```

Comment:
Use a branch while https://github.com/lekemula/solargraph-rspec/pull/30 is resolved.